<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewAllBankAccounts</class>
 <widget class="QWidget" name="ViewAllBankAccounts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>680</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>680</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ViewAllBankAccounts</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* فرم کلی */
QWidget {
    background-color: #2c3e50;
    font-family: &quot;Segoe UI&quot;, sans-serif;
    color: #ecf0f1;
}

/* عنوان */
QLabel {
    font-size: 20px;
    font-weight: bold;
    color: white;
}

/* دکمه خروج */
QPushButton#logoutButton {
    background-color: #e74c3c;
    color: white;
    font-size: 14px;
    padding: 8px 16px;
    border: none;
    border-radius: 8px;
}

QPushButton#logoutButton:hover {
    background-color: #c0392b;
}

/* تب‌ها */
QTabWidget::pane {
    border: 2px solid #3498db;
    border-radius: 10px;
    padding: 5px;
}

QTabBar::tab {
    background: #2980b9;
    color: white;
    padding: 8px 14px;
    border-top-left-radius: 8px;
    border-top-right-radius: 8px;
    margin-right: 2px;
}

QTabBar::tab:selected {
    background: #1abc9c;
    font-weight: bold;
}

/* جدول‌ها */
QTableWidget {
    background-color: #ecf0f1;
    color: #2c3e50;
    gridline-color: #bdc3c7;
    border-radius: 8px;
}

QHeaderView::section {
    background-color: #34495e;
    color: white;
    font-weight: bold;
    padding: 6px;
    border: none;
}

QTableWidget QTableCornerButton::section {
    background-color: #34495e;
}

QTableWidget::item {
    padding: 6px;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QPushButton" name="logoutButton">
     <property name="text">
      <string>Back To Dashboard</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="LL">
     <property name="styleSheet">
      <string notr="true">QLabel {
    font-size: 15px;
    font-weight: bold;
    color: rgb(0, 255, 234);
    padding: 10px 0;
    border-bottom: 2px solid #3498db;
    letter-spacing: 1px;
}
</string>
     </property>
     <property name="text">
      <string>All Bank Accounts</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::TabPosition::North</enum>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Qarz_al_Hasan</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableWidget_2"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Checking</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableWidget"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Savings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableWidget_3"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
